src/CarbonAware.CLI/src/Extensions/AnsiColorExtensions.cs

```diff
@@ -0,0 +1,49 @@
+namespace CarbonAware.CLI.Extensions
+{
+    public static class AnsiColorExtensions
```

Comment:
hmm, not sure if this would work with a non-ansi terminals. (see NO_COLOR env var). Maybe a nice to have for future?

Reply:
Good call.  Will remove for now and capture elsewhere with more flexibility around switching whether or not to apply.